src/dmd/dsymbol.d

```diff
@@ -2114,6 +2114,22 @@ extern (C++) final class DsymbolTable : RootObject
         return s;
     }
 
+    debug
+    {
+        /**
+        dump the symbol table contents
+        */
+        void dump()
```

Comment:
This could probably be renamed to something like "printSymbolTable".

Reply:
seems redundant...
```
symbolTable.printSymbolTable
```

What's the argument for the redundancy?

Reply:
Sorry. I didn't realize it was a member.  Still "dump" is quite informal (slang in a way); perhaps "print"

Reply:
ok

Reply:
uh...it looks like the `print` function is already taken...switched it back to `dump` unless you have another name you'd prefer.

Reply:
What does `print()` currently do?  Maybe it should be overridden.

Reply:
ok, fine with me :)